picr22-enne-1-voistlust/segment_module/segment.c

```diff
@@ -0,0 +1,134 @@
+#include "Python.h"
+#include "numpy/arrayobject.h"
+
+unsigned char lookup_table[0x1000000];
+
+static PyObject* set_table (PyObject * Py_UNUSED(self), PyObject *args) {

```

Comment:
Are you using this file?

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it